### PR TITLE
PWN-8349 - Initial screen for changing device share

### PR DESCRIPTION
**Pull request by 1sakov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-8349 

## Description of Work

Made only UI. 
Have one thing left - margin for device cell. Couldn't find a smooth solution for adding it

don't worry about files count. most of them are renamings

## Screenshots (Optional)

Screenshot_2023-06-11-22-45-55-54_1f0007833e6a513c8db3660ceb9f0957

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/jupiter/ui/info/SwapInfoBottomSheet.kt [hunk 1]
```diff
@@ -19,7 +19,7 @@ import kotlinx.coroutines.flow.flowOn
 import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.merge
 import kotlinx.coroutines.flow.onEach
-import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.components.finance_block.MainCellModel
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/jupiter/ui/info/SwapInfoBottomSheet.kt:22]: but it's called Finance block everywhere in Figma, why MainCell?

File: app/src/main/java/org/p2p/wallet/settings/ui/devices/DeviceCellMapper.kt [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+        cells += createDeviceCell(currentDeviceName, buttonRes = R.string.devices_set_up)
+
+        cells += createSectionCell(R.string.devices_authorization_device)
+
+        cells += createDeviceCell(oldDeviceName, R.string.devices_make_sure_your_device)
+
+        cells += createFooterCell()
+
+        return@withContext cells
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/settings/ui/devices/DeviceCellMapper.kt:50]: no need for labeled @ return
```suggestion
        cells
```

or I can suggest doing buildList and calling this += createHeaderCell and etc

File: app/src/main/java/org/p2p/wallet/settings/ui/recovery/RecoveryKitPresenter.kt [hunk 3]
```diff
@@ -61,7 +59,9 @@ class RecoveryKitPresenter(
         }
     }
 
-    private fun fetchMetadata() {
+    private fun fetchMetadataIfNeeded() {
+//        if (seedPhraseProviderType == SeedPhraseSource.MANUAL) return
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/settings/ui/recovery/RecoveryKitPresenter.kt:63]: ?

File: ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/MainCellStyle.kt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 package org.p2p.uikit.components.finance_block
 
-enum class FinanceBlockStyle {
+enum class MainCellStyle {
     FINANCE_BLOCK, BASE_CELL
```
Comment (gslevinkov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/MainCellStyle.kt:4]: can we add some figma links to ui kit here to distinct between base_cell and finance block?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/settings/ui/devices/DeviceCellMapper.kt (hunk 2 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/settings/ui/devices/DeviceCellMapper.kt:50]: Adding the label is okay too. 
It helps to understand that the current function is wrapped into asynchronous thread

**Review by Davran331122 — APPROVED** [review]

**Pull request merged**